FLEDGE.md

```diff
@@ -201,7 +204,8 @@ The function gets called once for each candidate ad in the auction.  The argumen
       'adComponents': ['https://cdn.com/ad_component_of_bid',
                        'https://cdn.com/next_ad_component_of_bid',
                        ...],
-      'biddingDurationMsec': 12
+      'biddingDurationMsec': 12,
+      'wasmHelper': ... /* a WebAssembly.Module object based on interest group's biddingWasmHelperUrl */
```

Comment:
I believe wasmHelper was mainly intended to help with computing a bid – and if so, should it be passed to generateBid() function?

Reply:
@yoav: What exactly does joining the WICG mean?

@sbelov: Yes. Thank you. And this chunk is actually for generateBid, unlike the first portion which refers to the wrong function.

(I imagine we may want something like this for scoreAd as well, but one thing at a time).